### PR TITLE
Use justfile instead of makefile

### PR DESCRIPTION
**Pull request by grieve54706**
Use [casey/just](https://github.com/casey/just) for more features and modernization.

### CODE REVIEW

File: ibis-server/README.md [hunk 1]
```diff
@@ -61,32 +61,33 @@ vim .env
...
 - Python 3.11
-- Install `poetry` with version 1.7.1: `curl -sSL https://install.python-poetry.org | python3 - --version 1.7.1`
-- Install pre-commit hooks: `make pre-commit-install`
-- Execute `make install` to install the dependencies
-- Execute `make test` to run the tests
+- Install [poetry](https://github.com/python-poetry/poetry) with version 1.7.1: `curl -sSL https://install.python-poetry.org | python3 - --version 1.7.1`
+- Install [casey/just](https://github.com/casey/just)
+- Install [pre-commit](https://pre-commit.com) hooks: `just pre-commit-install`
+- Execute `just install` to install the dependencies
+- Execute `just test` to run the tests
```
Comment (goldmedal) [inline ibis-server/README.md:79]: Moving to `just` is a good improvement. However, there're many requirement tools before starting to develop this project. How about implementing a `Makefile` to install all of them?

Comment (goldmedal) [reply ibis-server/README.md:79]: As discussed offline, we leave this issue for the user to decide how to install the requirements themselves.

**Review by goldmedal — APPROVED** [review]
Thanks @grieve54706 -- nice improvement. LGTM

**Comment by paopa** [comment]
Nice to know this tool, I will consider using it in the AI service as well. 🤙🤙🤙

**Pull request merged**